sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveTable.scala

```diff
@@ -101,6 +101,19 @@ case class InsertIntoHiveTable(
     val tmpLocation = getExternalTmpPath(sparkSession, hadoopConf, tableLocation)
     val fileSinkConf = new FileSinkDesc(tmpLocation.toString, tableDesc, false)
 
+    tableDesc.getOutputFileFormatClassName match {
```

Comment:
Move the whole logics into `saveAsHiveFile`, which is being shared by `InsertIntoHiveDirCommand` and `InsertIntoHiveTable`. Both need these logics.